airbyte-integrations/connectors/source-netsuite-odbc/source_netsuite_odbc/discover_utils.py

```diff
@@ -0,0 +1,110 @@
+
+from airbyte_cdk.models import (
+    AirbyteStream,
+)
+import json
+from typing import Iterable, Mapping, Any
+from collections import defaultdict 
+
+
+# See https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_4407755805.html#SuiteAnalytics-Connect-System-Tables
+# for descriptions of the system tables used
+class NetsuiteODBCTableDiscoverer():
+    def __init__(self, cursor):
```

Comment:
let's rename cursor to db_cursor or db_connection or something like that to avoid the ambiguity with an Airbyte cursor